pkg/chunk/cache/background.go

```diff
@@ -32,8 +32,8 @@ type BackgroundConfig struct {
 
 // RegisterFlagsWithPrefix adds the flags required to config this to the given FlagSet
 func (cfg *BackgroundConfig) RegisterFlagsWithPrefix(prefix string, description string, f *flag.FlagSet) {
-	f.IntVar(&cfg.WriteBackGoroutines, prefix+"memcache.write-back-goroutines", 10, description+"How many goroutines to use to write back to memcache.")
-	f.IntVar(&cfg.WriteBackBuffer, prefix+"memcache.write-back-buffer", 10000, description+"How many key batches to buffer for background write-back.")
+	f.IntVar(&cfg.WriteBackGoroutines, prefix+"cache.write-back-concurrency", 10, description+"At what concurrency to write back to cache.")
```

Comment:
Have considered changing the prefix from `cache.` into `background.`? Two possible reasons:
1. In the YAML it's called `background:`
2. The resulting flags are something like `-store.chunks-cache.cache.write-back-concurrency` where we have `-cache.cache.` which looks a bit redundant

It's an option question, not a request for change.